### PR TITLE
[KEP-2731] Add docs for Kubelet OpenTelemetry tracing graduation

### PR DESCRIPTION
**Pull request by saschagrunert**
PR for https://github.com/kubernetes/enhancements/issues/2831

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [WIP] [KEP-2731] Add docs for Kubelet OpenTelemetry tracing graduation

**Comment by saschagrunert** [comment]
Ready for review.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -127,7 +127,8 @@ For a reference to old feature gates that are removed, please refer to
 | `KubeletPodResources` | `true` | Beta | 1.15 | |
 | `KubeletPodResourcesGetAllocatable` | `false` | Alpha | 1.21 | 1.22 |
 | `KubeletPodResourcesGetAllocatable` | `true` | Beta | 1.23 | |
-| `KubeletTracing` | `false` | Alpha | 1.25 | |
+| `KubeletTracing` | `false` | Alpha | 1.25 | 1.26 |
+| `KubeletTracing` | `false` | Beta | 1.27 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:131]:
```suggestion
| `KubeletTracing` | `true` | Beta | 1.27 | |
```

Comment (saschagrunert) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: Fixed as suggested :+1:

**Comment by saschagrunert** [comment]
@sallyom PTAL

**Review by sallyom — APPROVED** [review]
Thanks for adding this @saschagrunert 
LGTM

**Comment by nate-double-u** [comment]
Hi @saschagrunert could you resolve the merge conflict?

**Comment by saschagrunert** [comment]
@nate-double-u done :+1:

File: content/en/docs/concepts/cluster-administration/system-traces.md [hunk 2]
```diff
@@ -86,10 +86,7 @@ Enabled without a configured endpoint, the default OpenTelemetry Collector recei
 
 #### Enabling tracing in the kubelet
 
-To enable tracing, enable the `KubeletTracing`
-[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-on the kubelet. Also, provide the kubelet with a
-[tracing configuration](https://github.com/kubernetes/component-base/blob/release-1.25/tracing/api/v1/types.go).
+To enable tracing, apply the [tracing configuration](https://github.com/kubernetes/component-base/blob/release-1.25/tracing/api/v1/types.go).
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/system-traces.md:89]:
```suggestion
To enable tracing, apply the [tracing configuration](https://github.com/kubernetes/component-base/blob/release-1.27/tracing/api/v1/types.go).
```
?

For GA, we'll want to link to something other than source code (eg a reference page).

File: content/en/docs/concepts/cluster-administration/system-traces.md [hunk 3]
```diff
@@ -103,6 +100,21 @@ tracing:
   samplingRatePerMillion: 100
 ```
 
+If the `samplingRatePerMillion` is set to one million (`1000000`), then every
+span will be sent to the exporter.
+
+With the graduation of the feature to beta, the kubelet will collect spans from
```
Comment (sftim) [inline content/en/docs/concepts/cluster-administration/system-traces.md:106]:
```suggestion
The kubelet in Kubernetes v{{< skew currentVersion >}} collects spans from
```
?

**Comment by saschagrunert** [comment]
Applied the suggestions from @sftim

**Review by sftim — COMMENTED** [review]
LGTM for docs

**Comment by sftim** [comment]
@saschagrunert who can provide a tech review here?

**Comment by saschagrunert** [comment]
Mostly @sallyom, which already LGTMed. We could also ask @kubernetes/sig-instrumentation-leads for approval.

**Comment by sftim** [comment]
Cool

/lgtm
/approve

**Pull request merged**